.vscode/extensions.json

```diff
@@ -2,7 +2,6 @@
     // See http://go.microsoft.com/fwlink/?LinkId=827846
     // for the documentation about the extensions.json format
     "recommendations": [
-        "marlinfirmware.auto-build",
```

Comment:
This extension recommendation should not be removed.

Reply:
I think I have deleted the .vscode folder, that might have generated new files.